src/components/StatusBadge/StatusBadge.tsx

```diff
@@ -1,20 +1,36 @@
 import { Box, chakra, ChakraComponent, useStyleConfig } from "@chakra-ui/react";
 import React, { forwardRef } from "react";
 
-export const statusBadgeTypeArray = ["low", "medium", "high"] as const;
+export const statusBadgeLevelArray = ["low", "medium", "high"] as const;
+export type StatusBadgeLevels = typeof statusBadgeLevelArray[number];
+export const statusBadgeTypeArray = [
+  "informative",
+  "negative",
+  "neutral",
+  "positive",
+  "recommendation",
+  "warningHigh",
+  "warningLow",
+  "low",
```

Comment:
should low medium and high be included in this array? we don't see any examples with type having any of these values.

Reply:
Oddly, in the original component, the `level` prop was typed as `StatusBadgeTypes`.  The `type` prop did not exist yet.  In the updated component, I aligned the prop types with the prop names, so, for backward compatability, I left those value in the array.